dbos/__init__.py

```diff
@@ -31,7 +31,6 @@
     "WorkflowHandleAsync",
     "WorkflowStatus",
     "WorkflowStatusString",
-    "load_config",
     "get_dbos_database_url",
```

Comment:
Remove this as well. Anything that relies on `dbos-config.yaml` exclusively. The only use for this would be to get the URL in DBOS Cloud, and we should recommend doing that through the `DBOS_DATABASE_URL` environment variable.